### PR TITLE
Remove react-native-router-flux from all scenes

### PR DESCRIPTION
**Pull request by swansontec**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [x] Tested on small Android
- [ ] n/a

### CODE REVIEW

File: src/components/scenes/CreateWalletAccountSelectScene.js [hunk 1]
```diff
@@ -108,10 +108,6 @@ class CreateWalletAccountSelect extends React.Component<Props, State> {
     logEvent('ActivateWalletSelect')
   }
 
-  onBack = () => {
```
Comment (thehobbit85) [inline src/components/scenes/CreateWalletAccountSelectScene.js:111]: Can I assume this is unused anyway?

File: src/components/scenes/CreateWalletReviewScene.js [hunk 2]
```diff
@@ -43,7 +43,10 @@ class CreateWalletReviewComponent extends React.Component<Props, State> {
     Keyboard.dismiss()
   }
 
-  goToWalletList = () => Actions.popTo(WALLET_LIST_SCENE)
+  goToWalletList = () => {
```
Comment (thehobbit85) [inline src/components/scenes/CreateWalletReviewScene.js:46]: why not keep it `goToWalletList = () => this.props.navigation.navigate('walletListScene')`?
Is there a reason you don't want to return the value of navigate?

File: src/components/scenes/FioAddressSettingsScene.js [hunk 3]
```diff
@@ -55,24 +55,28 @@ class FioAddressSettingsComponent extends React.Component<Props, LocalState> {
...
     showToast(s.strings.fio_request_renew_ok_text)
-    Actions.pop()
+    navigation.goBack()
     if (refreshAfterRenew) {
       window.requestAnimationFrame(() => {
-        Actions.refresh({ fioAddressName, expiration: expiration ? new Date(expiration) : '' })
+        navigation.setParams({
+          fioWallet,
+          fioAddressName,
+          expiration
```
Comment (thehobbit85) [inline src/components/scenes/FioAddressSettingsScene.js:72]: What about converting expiration to a date like before: `new Date(expiration)`?

File: src/components/scenes/ScanScene.js [hunk 4]
```diff
@@ -48,10 +48,10 @@ export const LOGIN_QR = 'loginQR'
 
 export class Scan extends React.Component<Props> {
   componentDidUpdate(prevProps: Props) {
-    const { route } = this.props
+    const { navigation, route } = this.props
     const { data } = route.params
-    if (data !== prevProps.route.params.data && Actions.currentScene !== 'DrawerOpen') {
-      Actions.drawerClose()
+    if (data !== prevProps.route.params.data) {
```
Comment (thehobbit85) [inline src/components/scenes/ScanScene.js:53]: Why is it ok to remove the 'Actions.currentScene !== 'DrawerOpen'' without a replacement?

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
1. Resolve small conflict
2. Address comments
3. Don't merge it yet. Even though it's a pretty "simple" PR, it's still very long and include changes to every scene in the app. I'd rather wait until we get the current staging (2.2.0) into master before we merge this so we can avoid any conflicts if we need to merge/cherry pick anything from develop.

File: src/components/scenes/CreateWalletAccountSelectScene.js (hunk 1 above)
Comment (swansontec) [reply src/components/scenes/CreateWalletAccountSelectScene.js:111]: Right, nobody every called this guy.

File: src/components/scenes/ScanScene.js (hunk 4 above)
Comment (swansontec) [reply src/components/scenes/ScanScene.js:53]: Because `Actions.currentScene !== 'DrawerOpen'`  is always `true`.

**Review by swansontec — COMMENTED** [review]
Found one more thing.

**Pull request merged**